### ISSUE TITLE
Cross-compilation guide partially outdated

### ISSUE THREAD

**Issue by anieuwland** [gtk-rs/gtk-rs.github.io#180]
The very useful cross-compilation [guide](https://gtk-rs.org/docs-src/tutorial/cross) needs some extra instructions. Gettext [requires](https://docs.rs/crate/gettext-sys/0.19.9/source/build.rs) setting an environment variable pointing to the mingw installation. In the Compiling section, right before the build command, it is necessary to do `export MINGW_PREFIX=$GTK_INSTALL_PATH`.

Unfortunately, `GTK_INSTALL_PATH` is only introduced in a later section (Packaging). The guide might need a little restructuring to stay coherent. I also think it would be valuable to mention for example on Fedora the relevant path is `/usr/x86_64-w64-mingw32/sys-root/mingw/`. 

Lastly, I think *think* (but am not sure) there is a mistake in `export PKG_CONFIG_PATH=/usr/i686-w64-mingw32/lib/pkgconfig`. The whole guide is about x86_64, but here we point to i686 data, can that be right? 

So to summarize:

* The compiling section needs to mention setting MINGW_PREFIX to satisfy gettext. 
* It might mistakenly refer to i686 instead of x86_64 for mingw's pkg_config.
* The document might need a little rewriting to stay coherent and mention path equivalents from other distributions (notably Fedora).

What is the process to make this happen, if I'd like to?

**Comment by sdroege** [comment]
You could create a PR that changes [this file](https://github.com/gtk-rs/gtk-rs.github.io/blob/master/docs-src/tutorial/cross.md). Thanks for noticing this :)
